tasks/Resources/RHODS_OLM/uninstall/uninstall.robot

```diff
@@ -19,7 +19,16 @@ Uninstall RHODS In OSD
   Trigger RHODS Uninstall 
     
 Uninstall RHODS In PSI
-  Fail  Not implemented yet
+      ${return_code}    ${output}	  Run And Return Rc And Output   git clone ${RHODS_INSTALL_REPO}
+      Log   ${output}
+      Should Be Equal As Integers	${return_code}	 0
+      ${git_folder} =  Get Regexp Matches    ${output}	   Cloning into \'(.*?)\'    1
+      Log   ${git_folder}[0]
+      ${return_code}    ${output}    Run And Return Rc And Output   (cd ${git_folder}[0]; ./cleanup.sh); wait $!; sleep 60   #robocop:disable
+      Should Be Equal As Integers	${return_code}	 0
+      ${return_code}	  ${output}    Run And Return Rc And Output   rm -rf ${git_folder}[0]
+      Log    ${output}
+      Should Be Equal As Integers	  ${return_code}	 0
 
 RHODS Operator Should Be Uninstalled
   Verify RHODS Uninstallation
```

Comment:
I know this is not part of this PR, but this keyword needs a fix, because it waits way longer than necessary after RHODS is uninstalled

Reply:
I have the fix in mind but it has to be fixed in the openshiftLibrary in the next sprint. Just now I did a workaround in this PR, The installation takes between 1 and 2 minutes

Reply:
great, sounds good. Which lines implements the workaround?

Reply:
`(cd ${git_folder}[0]; ./cleanup.sh); wait $!; sleep 60 ` This line of code execute the script cleanup.sh and wait for the subterminal finish. After that I did an sleep of 60 seconds, time enough to get the projects deleted when the verification check if the project exists the projects doesn't exists and verify the uninstallation.

Reply:
You can try in a PSI cluster with this command `run_robot_test.sh --test-case ./tasks/Tasks/rhods_olm.robot --include uninstall --test-variable cluster_type:PSI`